### DISCUSSION TITLE
Hello, Is this possible to use with remoteapp from linux?

### DISCUSSION THREAD

**Discussion by Progra-MMO** [FreeRDP/FreeRDP#10586] [Ideas]
I am in the need of use a remote application from linux (powerfull home server) on a work computer (cheap dell with windows where also can not install unaproved software. The possible solution that I tailored so far is run windows on a docker container and run remoteapp https://github.com/kimmknight/remoteapptool from there with the applications, but I am wondering if it's possible to run directly from linux.
In example, I have installed Brave Browser on Fedora, and I need to use it through an RDP or RemoteApp connection from Windows.

Thank you for your amazin job !

**Comment by akallabeth** [comment]
currently no.
(that is you want to run `mstsc` on windows and the remote app would be running on your linux server and the window shown on your windows machine, right?)

**Reply by Progra-MMO** [reply]
Right, exactly that.

Also found this software that could do the job but if I can't install anything on the computer
https://wiki.x2go.org/

**Reply by akallabeth** [reply]
you can simply run a normal RDP desktop session with dynamic resizing enabled.
Not as comfortable as direct per application windows but usable.
